examples/Cargo.toml

```diff
@@ -5,8 +5,8 @@ authors = ["maik klein <maikklein@googlemail.com>"]
 edition = "2018"
 
 [dependencies]
-winit = "0.25.0"
-image = "0.10.4"
+winit = "0.26.1"
+image = "0.23.14"
```

Comment:
Do we need these minimum patch versions? Otherwise just omit them entirely, this is just an example.